### DISCUSSION TITLE
Brew formula for terraform@0.14

### DISCUSSION THREAD

**Discussion by RayBB** [Homebrew/discussions#1371] [Everyday usage] [answered]
Hello,

Right now there are formula for terraform@0.11, terraform@0.12, terraform@0.13, and terraform@0.15.

However, there is not one for terraform@0.14

This is a problem because terraform doesn't let you upgrade directly from 0.13 -> 0.15 it requires you to step up each version. Using brew to install the various versions makes it easy to do the upgrade.
https://www.terraform.io/upgrade-guides/0-14.html

Is there any plan to add 0.14 terraform to brew?

Thanks

**Comment by SMillerDev** [comment, marked as answer]
https://github.com/Homebrew/homebrew-core/pull/75981
